src/pages/documentation/release-notes/index.md

```diff
@@ -4,6 +4,26 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## Aug 7, 2023
+
+### Roku SDK 1.0.0
+
+This is the first stable release of the Roku SDK, which exposes APIs to help developers to integrate the Adobe Experience Platform solutions with the Roku channel.
+
+Features:
+
+* **Adobe Experience Edge Network Integration:** Seamlessly send data to the Adobe Experience Edge Network. Optionally include an Experience Edge response callback using the API for greater control over data interactions.
+
+* **Automated ECID and Implementation Attachment:** Experience more efficient tracking and integration as ECID (Experience Cloud ID) and Implementation details are now automatically appended to every XDM Experience Event request.
+
+* **Custom Identity Synchronization:** Personalize user experiences effectively by syncing Custom Identities with each XDM Experience Event request. Unlock the potential of the Adobe Experience Edge Network while maintaining tailored interactions.
+
+For more information, refer to the documentation below:
+
+* [Getting Started](https://github.com/adobe/aepsdk-roku/blob/main/Documentation/getting-started.md)
+* [API Reference](https://github.com/adobe/aepsdk-roku/blob/main/Documentation/api-reference.md)
+* [Sample App](https://github.com/adobe/aepsdk-roku/blob/main/sample/simple-videoplayer-channel/README.md)
```

Comment:
for the sample app can we just point to the sample folder instead so in case we rename or add other samples in the future the link does not break

Reply:
I'd say not to do that. We have two apps under the sample folder, and the other one is only for the Roku SDK development setup. So it's intentional to this link to show the right sample app.